vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -229,13 +229,13 @@ func (feeder *clusterStateFeeder) setVpaCheckpoint(checkpoint *vpa_types.Vertica
 	vpaID := model.VpaID{Namespace: checkpoint.Namespace, VpaName: checkpoint.Spec.VPAObjectName}
 	vpa, exists := feeder.clusterState.Vpas[vpaID]
 	if !exists {
-		return fmt.Errorf("cannot load checkpoint to missing VPA object %+v", vpaID)
+		return fmt.Errorf("cannot load checkpoint to missing VPA object %s/%s", vpa.ID.Namespace, vpa.ID.VpaName)
```

Comment:
While you're here, does it make sense to also capitilize the error message? (ref: https://github.com/kubernetes/autoscaler/issues/6905)

Reply:
This particular case is not a log entry but it is an error. Error strings should are not capitalized in idiomatic go: see https://github.com/kubernetes/autoscaler/issues/6906.

Let's leave the logging improvements in another change. I didn't want to add too much other changes in this PR to make the review process easier.

Reply:
🤦 Yup, you're right. My bad!